src/modules/user/validation/userValidations.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response, NextFunction } from "express";
+import Joi from "joi";
+import httpStatus from "http-status";
+
+const userStatusSchema = Joi.object({
+  status: Joi.string().valid("enabled", "disabled").required().messages({
+    "string.base": "Status must be a string",
+    "any.only": "Status must be either 'enabled' or 'disabled'",
+    "any.required": "Status is required"
+  })
+});
+
+
+export const validateUserStatus = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
We already have middleware function to handle schemas. So, why this then

Reply:
resolved